ci-operator/jobs/openshift/origin/openshift-origin-main-presubmits.yaml

```diff
@@ -1445,9 +1445,10 @@ presubmits:
         - --gcs-upload-secret=/secrets/gcs/service-account.json
         - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
         - --lease-server-credentials-file=/etc/boskos/credentials
+        - --multi-stage-param=SHARD_ARGS="--shard-count 2 --shard-id 1"
         - --report-credentials-file=/etc/report/credentials
         - --secret-dir=/secrets/ci-pull-credentials
-        - --target=e2e-aws-ovn-serial-1of2
+        - --target=e2e-aws-ovn-serial
```

Comment:
What is target used for? Does it cause any problem to not match the job name?

Reply:
it references the `test` in the ci-operator config. In this case, it cannot match the job name. I believe I have fixed the issues that resulted from that already in https://github.com/openshift/ci-tools/pull/4564